src/ipolatev.f90

```diff
@@ -527,8 +541,10 @@ END SUBROUTINE IPOLATEV_GRIB2
   !!   SPECTRAL| 4 | 0,-1         | 12.60
   !!   N-BUDGET| 6 | -1,-1        | 0.33
   !!
-  SUBROUTINE IPOLATEV_grib1(IP,IPOPT,KGDSI,KGDSO,MI,MO,KM,IBI,LI,UI,VI, &
-       NO,RLAT,RLON,CROT,SROT,IBO,LO,UO,VO,IRET) bind(c)
+  !! @date July 2021
+  !! @author Kyle Gerheiser
+  subroutine ipolatev_grib1(ip,ipopt,kgdsi,kgdso,mi,mo,km,ibi,li,ui,vi, &
```

Comment:
I would not lower-case the code. Was this intentional?

Try and keep code changes to a minimum, or have no code changes at all, for the documentation tasks. That makes them easy to review. The first thing I look at in mine or @Hang-Lei-NOAA 's is that no lines of code changed. That way we know we are not breaking anything.

Reply:
Ok, I'll avoid doing that. It was intentional because the Doxygen params take the case of the variables in the function declaration.